### PR TITLE
Database Pool: Update images dependent on ubuntu-base and remove no longer used images

### PR DESCRIPTION
**Pull request by mpowaga**
Summary of changes:

- Updated ubuntu-base/9.2.x to ubuntu-base/9.3.0
- Removed mysql image
- Removed mssql image
- Updated readme.md

### CODE REVIEW

File: elixir/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM codesignal/erlang:v9.2.0
+FROM codesignal/erlang:v9.3.0
```
Comment (salbert96) [inline elixir/Dockerfile:1]: For the first layer we should update only the images that are based on `codesignal/ubuntu-base`
We don't have `codesignal/erlang:v9.3.0` yet, we need to update that layer after merging this and creating a release.

File: coffeescript/Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM codesignal/js:v9.2.1
+FROM codesignal/js:v9.3.0
```
Comment (salbert96) [inline coffeescript/Dockerfile:1]: Same problem here.

**Review by salbert96 — CHANGES_REQUESTED** [review]
@mpowaga I did not write in every place, but we should only update the first layer that is based on ubuntu-base (that's the reason why the ci fails).

File: erlang/Dockerfile [hunk 3]
```diff
@@ -1,4 +1,4 @@
-FROM codesignal/ubuntu-base:v9.3.0
```
Comment (salbert96) [inline erlang/Dockerfile]: I thinks this should be updated

Comment (mpowaga) [reply erlang/Dockerfile]: Good catch!

**Comment by salbert96** [comment]
@mpowaga seems like Julia does not have the last version, is it intentional?

**Comment by mpowaga** [comment]
> seems like Julia does not have the last version, is it intentional?

@salbert96 It fails to build. I will update it separately to investigate why it fails.

**Comment by mpowaga** [comment]
I'll create another PR that will update second layer images after new images are pushed to docker hub.

**Comment by salbert96** [comment]
> It fails to build. I will update it separately to investigate why it fails.

👍  we don't have to do it soon, feel free to create an issue and it can be done later, after the database-pool, to not block us.

**Review by salbert96 — APPROVED** [review]
🚢

File: coffeescript/Dockerfile (hunk 2 above)
Comment (salbert96) [reply coffeescript/Dockerfile:1]: @mpowaga seems like this was updated from `js`. Not a big deal, but we should make sure we build everything that is based on js.

**Pull request merged**